### PR TITLE
Fix Docstring errors in init.py

### LINKED ISSUE #112596: Fix docstring errors in init.py
Please fix the following issues. To test locally, run: 
 ```
pydocstyle path-to-file --count
```
Run this command before and after fixing the errors. This command prints out the number of errors at the end of the output. Paste the output to your PR description with the number before and after. 
- **File**: `torch/nn/init.py`, **Entity**: `calculate_gain`, **Line**: 68, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `uniform_`, **Line**: 123, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `uniform_`, **Line**: 123, **Description**: First line should end with a period (not 'm')
- **File**: `torch/nn/init.py`, **Entity**: `uniform_`, **Line**: 123, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `normal_`, **Line**: 141, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `normal_`, **Line**: 141, **Description**: First line should end with a period (not 'l')
- **File**: `torch/nn/init.py`, **Entity**: `normal_`, **Line**: 141, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `trunc_normal_`, **Line**: 165, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `trunc_normal_`, **Line**: 165, **Description**: First line should end with a period (not 'd')
- **File**: `torch/nn/init.py`, **Entity**: `trunc_normal_`, **Line**: 165, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `constant_`, **Line**: 187, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `ones_`, **Line**: 203, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `zeros_`, **Line**: 216, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `eye_`, **Line**: 229, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `eye_`, **Line**: 229, **Description**: First line should end with a period (not 'y')
- **File**: `torch/nn/init.py`, **Entity**: `eye_`, **Line**: 229, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `dirac_`, **Line**: 249, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `dirac_`, **Line**: 249, **Description**: First line should end with a period (not 'c')
- **File**: `torch/nn/init.py`, **Entity**: `dirac_`, **Line**: 249, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `xavier_uniform_`, **Line**: 311, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `xavier_uniform_`, **Line**: 311, **Description**: First line should end with a period (not 'd')
- **File**: `torch/nn/init.py`, **Entity**: `xavier_uniform_`, **Line**: 311, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `xavier_normal_`, **Line**: 338, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `xavier_normal_`, **Line**: 338, **Description**: First line should end with a period (not 'd')
- **File**: `torch/nn/init.py`, **Entity**: `xavier_normal_`, **Line**: 338, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `kaiming_uniform_`, **Line**: 376, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `kaiming_uniform_`, **Line**: 376, **Description**: First line should end with a period (not 'd')
- **File**: `torch/nn/init.py`, **Entity**: `kaiming_uniform_`, **Line**: 376, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `kaiming_normal_`, **Line**: 425, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `kaiming_normal_`, **Line**: 425, **Description**: First line should end with a period (not 'd')
- **File**: `torch/nn/init.py`, **Entity**: `kaiming_normal_`, **Line**: 425, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `orthogonal_`, **Line**: 462, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `orthogonal_`, **Line**: 462, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/init.py`, **Entity**: `orthogonal_`, **Line**: 462, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
- **File**: `torch/nn/init.py`, **Entity**: `sparse_`, **Line**: 507, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/init.py`, **Entity**: `sparse_`, **Line**: 507, **Description**: First line should end with a period (not 'e')
- **File**: `torch/nn/init.py`, **Entity**: `sparse_`, **Line**: 507, **Description**: First line should be in imperative mood (perhaps 'Fill', not 'Fills')

cc @carljparker

### PR DESCRIPTION
**Pull request by stanleyedward**
Fixes #112596 

Fix docstring errors in init.py

### Before the change -> 38 errors
```
╭─user@pc ~/Path/to/pytorch  ‹fix/docstring_init› 
╰─➤  pydocstyle torch/nn/init.py --count                                                                                                                                             127 ↵
torch/nn/init.py:1 at module level:
        D100: Missing docstring in public module
torch/nn/init.py:68 in public function `calculate_gain`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:123 in public function `uniform_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:123 in public function `uniform_`:
        D400: First line should end with a period (not 'm')
torch/nn/init.py:123 in public function `uniform_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:141 in public function `normal_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:141 in public function `normal_`:
        D400: First line should end with a period (not 'l')
torch/nn/init.py:141 in public function `normal_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:165 in public function `trunc_normal_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:165 in public function `trunc_normal_`:
        D400: First line should end with a period (not 'd')
torch/nn/init.py:165 in public function `trunc_normal_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:187 in public function `constant_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:203 in public function `ones_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:216 in public function `zeros_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:229 in public function `eye_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:229 in public function `eye_`:
        D400: First line should end with a period (not 'y')
torch/nn/init.py:229 in public function `eye_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:249 in public function `dirac_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:249 in public function `dirac_`:
        D400: First line should end with a period (not 'c')
torch/nn/init.py:249 in public function `dirac_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:311 in public function `xavier_uniform_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:311 in public function `xavier_uniform_`:
        D400: First line should end with a period (not 'd')
torch/nn/init.py:311 in public function `xavier_uniform_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:338 in public function `xavier_normal_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:338 in public function `xavier_normal_`:
        D400: First line should end with a period (not 'd')
torch/nn/init.py:338 in public function `xavier_normal_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:376 in public function `kaiming_uniform_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:376 in public function `kaiming_uniform_`:
        D400: First line should end with a period (not 'd')
torch/nn/init.py:376 in public function `kaiming_uniform_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:425 in public function `kaiming_normal_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:425 in public function `kaiming_normal_`:
        D400: First line should end with a period (not 'd')
torch/nn/init.py:425 in public function `kaiming_normal_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:462 in public function `orthogonal_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:462 in public function `orthogonal_`:
        D400: First line should end with a period (not 's')
torch/nn/init.py:462 in public function `orthogonal_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
torch/nn/init.py:507 in public function `sparse_`:
        D205: 1 blank line required between summary line and description (found 0)
torch/nn/init.py:507 in public function `sparse_`:
        D400: First line should end with a period (not 'e')
torch/nn/init.py:507 in public function `sparse_`:
        D401: First line should be in imperative mood (perhaps 'Fill', not 'Fills')
38
```

### After the change -> 0 errors
```
╭─user@pc ~/Path/to/pytorch  ‹fix/docstring_init*› 
╰─➤  pydocstyle torch/nn/init.py --count
0
```

cc @svekars @carljparker

### CODE REVIEW

**Comment by stanleyedward** [comment]
Added the suggested changes

**Review by mikaylagawarecki — APPROVED** [review]

**Comment by svekars** [comment]
Can you please run `lintrunner -a` locally to fix lint errors?

**Comment by stanleyedward** [comment]
> Can you please run `lintrunner -a` locally to fix lint errors?

applied the lintrunner patches in the last commit

File: torch/nn/init.py [hunk 1]
```diff
@@ -422,10 +432,11 @@ def kaiming_uniform_(
 def kaiming_normal_(
     tensor: Tensor, a: float = 0, mode: str = 'fan_in', nonlinearity: str = 'leaky_relu'
 ):
-    r"""Fills the input `Tensor` with values according to the method
-    described in `Delving deep into rectifiers: Surpassing human-level
-    performance on ImageNet classification` - He, K. et al. (2015), using a
-    normal distribution. The resulting tensor will have values sampled from
+    r"""Fill the input `Tensor` with values using a normal distribution.
```
Comment (kit1980) [inline torch/nn/init.py:435]: The first docstring line should include "kaiming" or something like that.
As is, the first line is exactly the same as for `xavier_normal_`, which is not good.

File: torch/nn/init.py [hunk 2]
```diff
@@ -422,10 +432,11 @@ def kaiming_uniform_(
...
+
+    Described in `Delving deep into rectifiers: Surpassing human-level
```
Comment (kit1980) [inline torch/nn/init.py:437]: What's described? "The method" probably.

**Review by kit1980 — COMMENTED** [review]
`xavier` and `kaiming` variants should have different first docstring lines, mentioning the method.

**Pull request closed without merging**